tests/model_serving/model_server/private_endpoint/utils.py

```diff
@@ -0,0 +1,121 @@
+import shlex
+import base64
+from typing import Optional, Any, Generator
+from urllib.parse import urlparse
+from contextlib import contextmanager
+
+from ocp_resources.pod import Pod
+from ocp_resources.deployment import Deployment
+from kubernetes.dynamic.client import DynamicClient
+from kubernetes.dynamic.exceptions import ResourceNotFoundError, ResourceNotUniqueError
+from ocp_resources.inference_service import InferenceService
+from pyhelper_utils.shell import run_command
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+class ProtocolNotSupported(Exception):
+    def __init__(self, protocol: str):
+        self.protocol = protocol
+
+    def __str__(self) -> str:
+        return f"Protocol {self.protocol} is not supported"
+
+
+class InvalidStorageArgument(Exception):
+    def __init__(
+        self,
+        storageUri: Optional[str],
+        storage_key: Optional[str],
+        storage_path: Optional[str],
+    ):
+        self.storageUri = storageUri
+        self.storage_key = storage_key
+        self.storage_path = storage_path
+
+    def __str__(self) -> str:
+        msg = f"""
+            You've passed the following parameters:
+            "storageUri": {self.storageUri}
+            "storage_key": {self.storage_key}
+            "storage_path: {self.storage_path}
+            In order to create a valid ISVC you need to specify either a storageUri value
+            or both a storage key and a storage path.
+        """
+        return msg
+
+
+def get_flan_deployment(client: DynamicClient, namespace: str, name_prefix: str) -> Deployment:
```

Comment:
There is nothing specific to the flan model in this fixture, right? Would it make sense to call it `get_kserve_predictor_deployment` or similar?